waspc/data/Generator/templates/react-app/src/DefaultLoader.tsx

```diff
@@ -0,0 +1,37 @@
+import { styled, keyframes } from 'wasp/core/stitches.config'
+
+const fullRotationKeyframes = keyframes({
+  '0%': { transform: 'rotate(0deg)' },
+  '100%': { transform: 'rotate(360deg)' }
+});
+
+// TODO: Do I need to add !important on all of this stuff because of Tailwind css?
```

Comment:
@infomiho any idea, do I need to add !important here?

Reply:
Yep, you should enable Tailwind for some test project and see what happens :) 

On the topic of which loader to pick, here are some example from ShadCN, NextUI, Chakra and DaisyUI

https://github.com/wasp-lang/wasp/assets/2223680/ac8d4f98-e952-4834-8207-23b9eec432da


https://github.com/wasp-lang/wasp/assets/2223680/b6f616aa-53c6-4db7-9703-e664a5bc11f0


https://github.com/wasp-lang/wasp/assets/2223680/108e1781-af0f-4e2b-ab21-56245b53605e


https://github.com/wasp-lang/wasp/assets/2223680/af78d614-fd42-4ea5-a23c-edd23a1b4636

They are all using a "single piece of a circle spinning" which I think we should also go for! So our aesthetic is as modern as possible.

Reply:
Here's a PR with the Chakra UI https://github.com/wasp-lang/wasp/pull/1990